### PR TITLE
Remove visible cCc text

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
https://trello.com/c/cgQAWlov/312-replace-ccc-placeholder-copy

## Description

- Replace cCc placeholders in readiness for Alpha testing
-

@neontribe/sea-map

### CODE REVIEW

File: src/Copy/Text.elm [hunk 1]
```diff
@@ -11,7 +11,7 @@ t : Key -> String
 t key =
     case key of
         SiteTitle ->
-            "[cCc] SEA MAP"
+            "Economic abuse support tool"
```
Comment (Nikomus) [inline src/Copy/Text.elm]: I would title case this.. but one for when we have a real title.

Comment (Nikomus) [reply src/Copy/Text.elm]: There will be a [cCc] in the index.html file as well. I think we found before that it flickers as the Elm app is loaded - also not too important to fix now.

**Review by Nikomus — DISMISSED** [review]
I think OK for now! I will reapprove if you want to do the index file 🌳

File: src/Copy/Text.elm (hunk 1 above)
Comment (katjam) [reply src/Copy/Text.elm]: Good catch. Done.

**Review by Nikomus — APPROVED** [review]
Ace! 💯

**Pull request merged**